mlir/include/mlir/Dialect/MIOpen/LowerMIOpenOps.h

```diff
@@ -9883,6 +10974,11 @@ struct XdlopsGemmV2RewritePattern
                   loc, ilnkb.create<MulIOp>(loc, ilnkiv, NConstantOp), laneId),
               ilnkb.create<MulIOp>(loc, NPerXdlopsConstantOp, olniv)));
 
+      if (KPack > 1)
```

Comment:
Why's this multiplication called for?

Reply:
With KPack, we are moving along LDS with stride in the multiples of KPack.